torchmetrics/regression/tweedie_deviance.py

```diff
@@ -0,0 +1,120 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Optional
+
+import torch
+from torch import Tensor
+
+from torchmetrics.functional.regression.tweedie_deviance import (
+    _tweedie_deviance_score_compute,
+    _tweedie_deviance_score_update,
+)
+from torchmetrics.metric import Metric
+
+
+class TweedieDevianceScore(Metric):
+    r"""
+    Computes the `Deviance Score <https://en.wikipedia.org/wiki/Tweedie_distribution#The_Tweedie_deviance>`_ between
+    targets and predictions:
+
+    .. math::
+        deviance\_score(\hat{y},y) =
```

Comment:
have you checked this renders correctly?

Reply:
Yes I had checked, it does render correctly.